### PR TITLE
add aliyun oss

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ Add aliyun oss to helm and operator backend]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/deploy/deploy-k8s-helm.md [hunk 1]
```diff
@@ -182,11 +182,17 @@ Please note that increasing the CPU resource will not automatically increase the
...
 
-For example, if you intend to use AWS S3 as the state backend, you can revise the configuration as follows:
 
-```bash
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="state_backend_options">
+
+<TabItem value="AWS S3" label="AWS S3">
```
Comment (CharlieSYH) [inline docs/deploy/deploy-k8s-helm.md]: Please use lowercase without empty space for the tab item value: value="s3".

File: docs/deploy/deploy-k8s-helm.md [hunk 2]
```diff
@@ -203,3 +209,27 @@ stateStore:
       accessKey: <your access key>
       secretAccessKey: <your secret access key>
 ```
+</TabItem>
+
+<TabItem value="aliyun-oss" label="Aliyun OSS">
```
Comment (CharlieSYH) [inline docs/deploy/deploy-k8s-helm.md]: Please check the brand name for Aliyun.
image

**Review by CharlieSYH — APPROVED** [review]

File: docs/deploy/risingwave-kubernetes.md [hunk 3]
```diff
@@ -369,6 +370,39 @@ spec:
...
+<TabItem value="alibaba-cloud-oss" label="Alibaba Cloud OSS">
+
+```yaml
+spec:
+  stateStore:
+    # Prefix to objects in the object stores or directory in file system. Default to "hummock".
+    dataDirectory: hummock
+    
+    # Declaration of the Alibaba Cloud OSS state store backend.
+    aliyunOSS:
```
Comment (WanYixian) [inline docs/deploy/risingwave-kubernetes.md:382]: Is this necessary to change it into Alibaba Cloud OSS? Others LGTM!

File: versioned_docs/version-1.6/deploy/risingwave-kubernetes.md [hunk 4]
```diff
@@ -367,6 +368,38 @@ spec:
...
+<TabItem value="alibaba-cloud-oss" label="Alibaba Cloud OSS">
+
+```yaml
+spec:
+  stateStore:
+    # Prefix to objects in the object stores or directory in file system. Default to "hummock".
+    dataDirectory: hummock
+    
+    # Declaration of the Alibaba Cloud OSS state store backend.
+    aliyunOSS:
```
Comment (WanYixian) [inline versioned_docs/version-1.6/deploy/risingwave-kubernetes.md:381]: Same above

**Review by WanYixian — APPROVED** [review]

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks for the prompt updates.

**Pull request merged**